code/modules/flufftext/Hallucination.dm

```diff
@@ -1318,3 +1319,33 @@ GLOBAL_LIST_INIT(hallucination_list, list(
 	else
 		to_chat(C,"<span class='userdanger'>[G] violently grabs you!</span>")
 	qdel(src)
+
+/datum/hallucination/naked
+	var/image/image
+
+/datum/hallucination/naked/New(mob/living/carbon/C, forced = TRUE)
+	set waitfor = FALSE
+	..()
+	if (C.client && C.client.prefs)
+		var/datum/preferences/prefs = C.client.prefs
+		var/mob/living/carbon/human/dummy/M = generate_or_wait_for_human_dummy("hallucination")
```

Comment:
Use defines.

Reply:
wut

Reply:
Add a define for this dummy in this section:
https://github.com/Citadel-Station-13/Citadel-Station-13/blob/92ead7e65d955f50c2dd877524515dc79b605e2f/code/__DEFINES/misc.dm#L404
And use it instead.